### DISCUSSION TITLE
Feature request: Possibility to select/unselect some info lines shown in cars display?

### DISCUSSION THREAD

**Discussion by 0utc45t** [BimmerGestalt/AAIdrive#783] [Ideas]
I have a -22 BMW i3 without REX. Would it be possible to have an option in AAIdrive phone app (I have android phone) to exclude the REX related information, so that it is not shown on the "detailed vehicle info" screen?

I see oil temperature fixed at 0 degrees Celsius and engine temperature fixed at 118 degrees Celsius in "detailed vehicle info" screen and those are totally useless (and IMO a bit annoying) in every i3 without REX engine. Those could be replaced with some other information because then there would be 4 lines free in that screen to show something else. Even better, if user could choose information to be shown in that or all info pages... user could "build" own info pages in the app by selecting the information user wants to see in each of those pages. It would be really nice addition to already awesome app.

The dev(s) who have made AAIdrive, I salute you! :-)

**Comment by hufman** [comment]
Good morning!
Those temperatures are interesting, my fully electric Mini Cooper shows the same values :)
Automatically hiding the values would be easy enough, if there's another data point that indicates whether the car has an engine or not. What data would you like to see instead?
I don't plan to add fully customized data screens currently, but would welcome contributions.

**Reply by 0utc45t** [reply]
Good evening!
I presume that those values come from the lack of the sensors, the cars computers input lines for those sensors has either pull up or pull down resistors and without sensors connected, AD converter gets constant voltage value and we see those conversion result fixed at the values already mentioned. I believe those values are a dead give away values for fully electric version of these cars, even more so because those values should never happen together at the same time in a real ICE. 
I had to took a drive to check thru what values there are on the other pages on the cars screen and screen captures of AAIdrive apps information. I was going to suggest that Acc Battery value could be shown and noticed that it actually is shown this time. Had to check the AAIdrive apps last update, as I thought first that you already had changed something, but no, the versions date is few months back. Now when I check thru those pics I noticed that I've already taken a pic earlier of the "detailed vehicle info" page... and on one pic that info is shown, on the other it is not... Weird. Well, I would keep that Acc Battery line there in any case and for the 2 remaining free lines I would add Interior and Exterior temperatures.
My computer language skills are limited to assembly, basic (C64 time, yihaa!) and bash. I believe those are not used in AAIdrive and I can not be of any assistance....
